nsv13/code/modules/overmap/fighters/_fighters.dm

```diff
@@ -1004,7 +1004,7 @@ due_to_damage: If the removal was caused voluntarily (FALSE), or if it was cause
 /obj/item/fighter_component/fuel_tank
 	name = "Fighter Fuel Tank"
 	desc = "The fuel tank of a fighter, upgrading this lets your fighter hold more fuel."
-	icon_state = "fueltank"
+	icon_state = "fueltank_tier1"
```

Comment:
we'll need to override the icon update proc then, or these will all be invisible because it will look for _tier1_tier1